routes/spec.js

```diff
@@ -1003,6 +1003,162 @@ Consider supporting The Slothpixel Project on Patreon to help cover the hosting
+                      type: 'array',
+                      items: {
+                        type: 'string',
+                      },
+                    },
+                    ranks: {
+                      type: 'array',
+                      items: {
+                        type: 'object',
+                        properties: {
+                          name: {
+                            type: 'string',
+                          },
+                          permissions: {
+                            type: 'array',
+                            items: {
+                              type: 'number',
+                            },
+                          },
+                          default: {
+                            type: 'boolean',
+                          },
+                          tag: {
+                            type: 'string',
+                          },
+                          created: {
+                            type: 'integer',
+                          },
+                          priority: {
+                            type: 'integer',
+                          },
+                        },
+                      },
+                    },
+                    members: {
+                      description: 'Array playerof players on the guild',
+                      type: 'array',
+                      items: {
+                        type: 'object',
+                        properties: {
+                          uuid: {
+                            description: 'Player UUID',
+                            type: 'string',
+                          },
+                          rank: {
+                            description: 'Player rank in the guild',
+                            type: 'string',
+                          },
+                          joined: {
+                            description: 'Member join date',
+                            type: 'integer',
+                          },
+                          quest_participation: {
+                            description: 'How many much the member has contributed to guild quests',
+                            type: 'integer',
+                          },
+                          exp_history: {
+                            description: 'Contains raw guild xp earned in the past week. Uses format YYYY-MM-DD.',
+                            type: 'object',
+                          },
+                          muted_till: {
+                            description: 'Date the member is muted until',
+                            type: 'integer',
+                          },
+                        },
+                      },
+                    },
+                  },
+                },
+              },
+            },
+          },
+        },
+        route: () => '/guilds/id/:id',
+        func: async (request, response, callback) => {
```

Comment:
Can you also add the `populatePlayers` parameter? https://github.com/slothpixel/core/blob/master/routes/spec.js#L839

Reply:
Also you need to add the parameter to the parameters array so it shows up on docs correctly

Reply:
I will when I get home :)

Reply:
I already did that.

Reply:
On lines 1015 and 859

Reply:
They are already there, unless something is messed up

Reply:
Are you sure you've commited them? I can't see them on github

Reply:
thats weird, lemme check github